integration_tests/tests/consistency/row_comparisons/consistency_sessions.sql

```diff
@@ -0,0 +1,46 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+-- this test ensures the daily_activity end model matches the prior version
+with prod as (
+    select *
+    from {{ target.schema }}_mixpanel_prod.mixpanel__sessions
+),
+
+dev as (
+    select *
+    from {{ target.schema }}_mixpanel_dev.mixpanel__sessions
+),
```

Comment:
The field `event_frequencies` is a result of a string agg array and the ordering of the fields can vary based on the run. Therefore, we should exclude this field from the comparison so we do not see false failures in the future.

```suggestion
{% set exclude_fields = ["event_frequencies"] %}
{% set fields = dbt_utils.star(from=ref('mixpanel__sessions'), except=exclude_fields) %}

-- this test ensures the daily_activity end model matches the prior version
with prod as (
    select {{ fields }}
    from {{ target.schema }}_mixpanel_prod.mixpanel__sessions
),

dev as (
    select {{ fields }}
    from {{ target.schema }}_mixpanel_dev.mixpanel__sessions
),
```

Reply:
Thank you--added!